research/src/components/file-anatomy.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import './anatomy.css'
+
+const FileAnatomy = () => {
+  return (
+    <div class="component-anatomy-wrapper">
+      <input type="checkbox" id="show-slots" />
+      <label for="show-slots"> Show slots</label>
+      <div class="component-anatomy">
+        <host name="input">
```

Comment:
we agreed in a telecon that we should have the Open UI components be that of what we would propose as the new element but under a web component namespace.

So let's change this to `oui-file`.

Reply:
Done, it's now `openui-file`.